REFERENCE.md

```diff
@@ -275,6 +275,9 @@ An optional name can be added, so that only process functions defined for that n
 **Note:**
 You need to define _processors_ for all the named jobs that you add to your queue or the queue will complain that you are missing a processor for the given job, unless you use the `*` as job name when defining the processor.
 
+**Note:**
+Considering all jobs in a terminal state (`failed` or `completed`) are stored in redis, depending on the number of jobs running and your redis setup, you might want to setup a default maximum number of jobs kept, using the `removeOnComplete` and `removeOnFailed` options when creating a queue so redis does not end up running out of memory.
```

Comment:
Instead of "terminal" it is better to use "finished" as this is the term we use internally.

Reply:
redis -> Redis